### PR TITLE
Feat anima lllite

### PR DESCRIPTION
**Pull request by kohya-ss**
An experimental implementation of ControlNet-LLLite for Anima.

This feature is experimental and may change. The hyperparameters are unknown. Community contributions and research are welcome.

The experimental ComfyUI node has been released as follows:
https://github.com/kohya-ss/ComfyUI-Anima-LLLite

The sample weight is uploaded here:
https://huggingface.co/kohya-ss/Anima-LLLite

> The architecture has been significantly updated. Previous sample weights are no longer usable.

The model was trained for 4 epochs with 2000 pairs of Anima-generated images and pseudo-lineart images. Batch size was 6, and the learning rate was 2e-4.

```
accelerate launch --num_cpu_threads_per_process 1 --mixed_precision bf16 anima_train_control_net_lllite.py --pretrained_model_name_or_path path/to/anima-preview3-base.safetensors --vae path/to/qwen_image_vae.safetensors --qwen3 path/to/qwen_3_06b_base.safetensors  --output_dir=path/to/anima_lora --mixed_precision bf16 --save_precision bf16 --optimizer_type adamw8bit --learning_rate 2e-4 --max_data_loader_n_workers 2 --persistent_data_loader_workers --dataset_config=path/to/anima-lineart-v1.toml --lllite_mlp_dim 32 --cond_emb_dim 32 --output_name=anima-lllite-test-1 --logging_dir ./logs --log_prefix anima-lllite-test-1- --max_train_epochs 4  --highvram --vae_batch_size 8 --save_every_n_epochs 1 --cache_text_encoder_outputs_to_disk --cache_latents_to_disk --attn_mode flash --save_model_as safetensors  --timestep_sampling shift --discrete_flow_shift 3.0 --seed 42
```

I used [tori29umai's LoRA for FramePack](https://note.com/tori29umai/n/n3447ca5b1437) to create the lineart images. Thank you very much.

### CODE REVIEW

**Comment by kohya-ss** [comment]
It seems the prompt needs to match the content of the control image to some extent.

Control image:
lineart1

Generated image with ComfyUI:
ComfyUI_00282_

**Comment by kohya-ss** [comment]
The architecture has been significantly updated. I have released four weights: line art, pose, depth map, and scribble.

File: networks/control_net_lllite_anima.py [hunk 1]
```diff
@@ -0,0 +1,892 @@
...
+            # patchify (/2) は DiT 内部 (prepare_embedded_sequence) で実施されるため、
+            # ここでは latent HW * 8 == cond_image HW を期待する。
+            # conditioning1 (stride 16) は cond_image を /16 = latent/2 = token 空間に揃える。
+            expected_h = x.shape[-2] * 8
+            expected_w = x.shape[-1] * 8
+            assert cond_image.shape[-2] == expected_h and cond_image.shape[-1] == expected_w, (
+                f"cond_image HW mismatch: latent={x.shape[-2]}x{x.shape[-1]} -> expected "
+                f"{expected_h}x{expected_w}, got {cond_image.shape[-2]}x{cond_image.shape[-1]}"
+            )
+            self.lllite.set_cond_image(cond_image)
```
[the remark this answers is not in the document]
Comment (kohya-ss) [reply networks/control_net_lllite_anima.py:481]: This is intended. Because if the wrapper is used for inference unexpectedly, forward is called without cond_image for denoising steps, it disables LLLite.

File: docs/anima_train_control_net_lllite.md [hunk 2]
```diff
@@ -0,0 +1,506 @@
+# ControlNet-LLLite Training Guide for Anima using `anima_train_control_net_lllite.py` / `anima_train_control_net_lllite.py` を用いた Anima モデルの ControlNet-LLLite 学習ガイド
```
[the remark this answers is not in the document]
Comment (kohya-ss) [reply docs/anima_train_control_net_lllite.md:1]: The original comment seems to be better.
